### PR TITLE
Update saos, sers, and eqs links to be https

### LINKED ISSUE #2661: Update EQS, SERS, and SAOS links to HTTPS
For security compliance, the below listed URLs will be updated to using strictly https in the near future.

1. http://eqs.fec.gov/eqs/searcheqs
2. http://sers.fec.gov/fosers/
3. http://saos.fec.gov/saos/searchao

**Completion criteria:**
- [ ] Coordinate with the PM for that application to see when the implementation will be going live so that we can push out our changes the same time
- [x] Confirm with content team about content linking to the above applications
- [ ] Confirm with legal application team to set up a redirect from http to https. Without this, a lot of content page links will break.
- [x] Make banner code changes to point to https

### PR DESCRIPTION
**Pull request by patphongs**
## Summary

- Resolves #2661 
_Updates saos, sers, and eqs links to point to https._

## Impacted areas of the application
List general components of the application that this PR will affect:

-  AOs landing: http://localhost:8000/data/legal/advisory-opinions/
- Enforcement landing: http://localhost:8000/legal-resources/enforcement/
- Meeting hearings tab: http://localhost:8000/meetings/?tab=hearings

### CODE REVIEW

File: fec/fec/static/js/legal/LegalSearch.js [hunk 1]
```diff
@@ -176,7 +176,7 @@ class LegalSearch extends React.Component {
                   using keywords, AO numbers, names of requestors and more. For
                   additional search filters, you can still search AOs using our
                   legacy{' '}
-                  <a href="http://saos.fec.gov/saos/searchao">AO search</a>.
+                  <a href="https://saos.fec.gov/saos/searchao">AO search</a>.
```
Comment (rfultz) [inline fec/fec/static/js/legal/LegalSearch.js]: For accessibility and SEO, what d'you think about adding a `title` parameter to the link? Something like `title="Search advisory opinions"`?

File: fec/home/templates/home/commission_meetings.html [hunk 2]
```diff
@@ -109,7 +109,7 @@ <h2>No results</h2>
           </section>
           <section id="meetings-hearings" role="tabpanel" aria-hidden="true" aria-labelledby="hearings">
             <h2>Public hearings</h2>
-            <p>The Commission periodically holds public hearings at its headquarters in Washington, DC. These hearings offer interested persons an opportunity to testify concerning proposed regulations and other matters that come before the Commission. This page lists upcoming public hearings as well as those held from 2005 through the present. Use <a href="http://sers.fec.gov/fosers/">SERS</a> to search for all archived rulemaking hearings.</p>
+            <p>The Commission periodically holds public hearings at its headquarters in Washington, DC. These hearings offer interested persons an opportunity to testify concerning proposed regulations and other matters that come before the Commission. This page lists upcoming public hearings as well as those held from 2005 through the present. Use <a href="https://sers.fec.gov/fosers/">SERS</a> to search for all archived rulemaking hearings.</p>
```
Comment (rfultz) [inline fec/home/templates/home/commission_meetings.html]: Is "SERS" common enough that search engines and screen readers will know what it is when they hear/see "SERS" without the content around it? Should we add a `title` parameter to the link?

File: fec/legal/templates/legal-advisory-opinions-landing.jinja [hunk 3]
```diff
@@ -51,7 +51,7 @@
       </div>
       <div class="row">
         <div class="message message--info">
-          <p>The advisory opinion search feature includes all FEC advisory opinions &#40;AOs&#41; to date. You can search all FEC AOs using keywords, AO numbers, names of requestors and more. For additional search filters, you can still search AOs using our legacy <a href="http://saos.fec.gov/saos/searchao">AO search</a>.</p>
+          <p>The advisory opinion search feature includes all FEC advisory opinions &#40;AOs&#41; to date. You can search all FEC AOs using keywords, AO numbers, names of requestors and more. For additional search filters, you can still search AOs using our legacy <a href="https://saos.fec.gov/saos/searchao">AO search</a>.</p>
```
Comment (rfultz) [inline fec/legal/templates/legal-advisory-opinions-landing.jinja]: For accessibility and SEO, what d'you think about adding a `title` parameter to the link? Something like `title="Search advisory opinions"`?

File: fec/legal/templates/partials/legal-search-results-advisory-opinion.jinja [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <div class="message message--info">
-  <p>The advisory opinion search feature includes all FEC advisory opinions &#40;AOs&#41; to date. You can search all FEC AOs using keywords, AO numbers, names of requestors and more. For additional search filters, you can still search AOs using our legacy <a href="http://saos.fec.gov/saos/searchao">AO search</a>.</p>
+  <p>The advisory opinion search feature includes all FEC advisory opinions &#40;AOs&#41; to date. You can search all FEC AOs using keywords, AO numbers, names of requestors and more. For additional search filters, you can still search AOs using our legacy <a href="https://saos.fec.gov/saos/searchao">AO search</a>.</p>
```
Comment (rfultz) [inline fec/legal/templates/partials/legal-search-results-advisory-opinion.jinja]: For accessibility and SEO, what d'you think about adding a `title` parameter to the link? Something like `title="Search advisory opinions"`?

File: fec/fec/static/js/legal/LegalSearch.js (hunk 1 above)
Comment (patphongs) [reply fec/fec/static/js/legal/LegalSearch.js]: Added title attribute

File: fec/home/templates/home/commission_meetings.html (hunk 2 above)
Comment (patphongs) [reply fec/home/templates/home/commission_meetings.html]: Spelled out SERS and added title attribute

File: fec/legal/templates/partials/legal-search-results-advisory-opinion.jinja (hunk 4 above)
Comment (patphongs) [reply fec/legal/templates/partials/legal-search-results-advisory-opinion.jinja]: Added title attribute

**Pull request merged**